packages/amplication-server/src/core/notifications/modules/inApp/inAppNotification.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, Inject, BadRequestException } from '@nestjs/common';
+import { IInAppNotification } from '../../contracts/inAppNotification.interface';
+import { Novu } from '@novu/node';
+import { WINSTON_MODULE_PROVIDER } from 'nest-winston';
+import { Logger } from 'winston';
+
+@Injectable()
+export class InAppNotificationService implements IInAppNotification {
```

Comment:
This can also be called a NotificationService since Novu handles the abstraction between multiple channels in case you would want send other channels with our workflow editor. But will def work like this aswell :)

Reply:
@scopsy yes. I thought so, but I wasn't sure. On the one hand, I created a template for email notification and saw that I would need to call the same method with the same parameters. But, on the other hand, I thought that we might need more business logic for SMS and email notifications that the in-app notifications shouldn't care of 🤷🏽‍♀️

And thank you for your review. Much appreciate that 🙂